bin/fastlane

```diff
@@ -91,7 +91,7 @@ class FastlaneApplication
       c.option '-f', '--force', 'Overwrite the existing README.md in the ./fastlane folder'
 
       c.action do |args, options|
-        ff = Fastlane::FastFile.new(File.join(Fastlane::FastlaneFolder.path || '.', 'Fastfile'))
+        ff = Fastlane::FastFile.new(Fastlane::FastlaneFolder.fastfile_path)
```

Comment:
This seems to be repeated everywhere, worth pulling into a method here?

e.g.

`def load_local_actions
  Fastlane::FastFile.new(Fastlane::FastlaneFolder.fastfile_path)
end